plugin/opencv/cv_api.h

```diff
@@ -0,0 +1,35 @@
+/*!
+ *  Copyright (c) 2016 by Contributors
+ * \file cv_api.h
+ * \brief C API for opencv
+ * \author Junyuan Xie
+ */
+#ifndef PLUGIN_OPENCV_CV_API_H_
+#define PLUGIN_OPENCV_CV_API_H_
+
+#include <mxnet/c_api.h>
+
+MXNET_DLL int MXCVImdecode(
+  const unsigned char *img,
+  const mx_uint len,
+  const int flag,
+  NDArrayHandle *out);
+
```

Comment:
can we make the API registerable as mxnet's ndarray API? So most cases there is no need in change of API interface and python